src/routes/safe/components/Transactions/TxList/MethodValue.tsx

```diff
@@ -49,6 +50,34 @@ const GenericValue = ({ method, type, value }: RenderValueProps): React.ReactEle
 }
 
 const Value = ({ type, ...props }: RenderValueProps): React.ReactElement => {
+  if (isNestedArrayParameter(type)) {
+    return (
+      <>
+        [
+        <NestedWrapper>
```

Comment:
Why are you nesting in two wrappers?

Reply:
They just add left padding. It's a styled component defined on top of the file.

Reply:
Makes sense. I would suggest we truncate the hash. In the screenshot it spreads across two lines.